test/specs/commonTests.js

```diff
@@ -239,9 +232,19 @@ export const isConformant = (Component, requiredProps = {}) => {
   // Handles className
   // ----------------------------------------
   describe('className (common)', () => {
-    it(`has the Stardust className "${sdClass}"`, () => {
-      render(<Component {...requiredProps} />)
-        .should.have.className(sdClass)
+    it('does not have an sd-* className', () => {
+      const wrapper = shallow(<Component {...requiredProps} />)
+      const className = wrapper.prop('className')
+      const children = wrapper.children()
+
+      // component itself
+      if (className) className.should.not.contain('sd-')
+
+      // children
+      children.forEach(c => {
+        const childClassName = c.prop('className')
+        if (childClassName) childClassName.should.not.contain('sd-')
+      })
```

Comment:
I've inverted this test for now to help catch any further sd-* classes from other PRs that have not yet been scrubbed.  After a while, we can remove this test.